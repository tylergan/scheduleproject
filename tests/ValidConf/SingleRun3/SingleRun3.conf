at 0900 run /usr/bin/python3 tests/ValidConf/SingleRun3/SingleRun3.py
at 1000 run /usr/bin/python3 tests/ValidConf/SingleRun3/SingleRun3.py