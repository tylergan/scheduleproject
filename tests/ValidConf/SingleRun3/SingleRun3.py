import sys

try:
    with open("file_does_not_exist", "r"):
        pass
except FileNotFoundError:
    print("The file was not found, but I will not exit this program with a status greater than 0")
