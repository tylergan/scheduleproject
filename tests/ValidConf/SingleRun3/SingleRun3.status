will run at XX XX XX 09:00:00 2020 /usr/bin/python3 tests/ValidConf/SingleRun3/SingleRun3.py
will run at XX XX XX 10:00:00 2020 /usr/bin/python3 tests/ValidConf/SingleRun3/SingleRun3.py
ran XX XX XX 09:00:00 2020 /usr/bin/python3 tests/ValidConf/SingleRun3/SingleRun3.py
will run at XX XX XX 10:00:00 2020 /usr/bin/python3 tests/ValidConf/SingleRun3/SingleRun3.py
ran XX XX XX 09:00:00 2020 /usr/bin/python3 tests/ValidConf/SingleRun3/SingleRun3.py
ran XX XX XX 10:00:00 2020 /usr/bin/python3 tests/ValidConf/SingleRun3/SingleRun3.py
