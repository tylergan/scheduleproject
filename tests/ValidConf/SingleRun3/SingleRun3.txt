Testing that runner.py and runstatus.py outputs the correct output to STDOUT/STDERR when there is:
    1) A single command line in the config file while trying to run a Python file

NOTE THAT:
You will notice that, despite an exception occurring, the program does not exit with a status code, hence its default returns 0, causing runner.py to say that the
process ran successfully (which it did for this program).