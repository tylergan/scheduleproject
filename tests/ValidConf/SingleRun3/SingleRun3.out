The file was not found, but I will not exit this program with a status greater than 0
The file was not found, but I will not exit this program with a status greater than 0
nothing left to run
